tests/integration/tables/process_open_pipes.cpp

```diff
@@ -0,0 +1,226 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+// Sanity check integration test for process_open_pipes
+// Spec file: specs/posix/process_open_pipes.table
+
+#include <fcntl.h>
```

Comment:
Do you mind grouping these includes (where a group is separated by a double newline)? 

There is no strict convention but 

osquery-includes

c-includes

this is mostly a nick-pick but it does provide value, if the osquery includes are grouped first then we have a stronger guarantee the order of includes is not implicitly satisfying a dependency.